frontend/src/components/Chat/ChatMessage.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-list-item class="messageItem">
+    <v-list-item-avatar>
+      <chat-message-menu
+        :targetId="senderId"
+        @chat-message-menu-selection="
+          $emit('chat-message-menu-selection', $event)
+        "
+      >
+        <v-img
+          max-height="45"
+          max-width="45"
+          :src="'https://static.generated.photos/vue-static/home/hero/3.png'"
+        >
+        </v-img>
+      </chat-message-menu>
+    </v-list-item-avatar>
+    <v-list-item-content>
+      <v-list-item-title
+        v-text="`${sender} -- ${formattedDate}`"
+      ></v-list-item-title>
+      <v-list-item-subtitle v-text="content"></v-list-item-subtitle>
```

Comment:
```suggestion
      <v-list-item-subtitle>
        {{ content }}
      </v-list-item-subtitle>
```
To improve legibility.